lms/templates/registration/activate_account_notice.html

```diff
@@ -6,12 +6,16 @@
 <div class="wrapper-msg urgency-high">
     <div class="msg">
         <div class="msg-content">
-            <h2 class="title">${_("Thanks for Registering!")}</h2>
+            <h2 class="title">${_("You're almost there!")}</h2>
             <div class="copy">
                 <p class='activation-message'>${Text(_(
-                        "You've successfully created an account on {platform_name}. We've sent an account "
-                        "activation message to {email_start}{email}{email_end}. To activate your account and start enrolling in "
-                        "courses, click the link in the message."
+                        "There's just one more step: Before you
```

Comment:
@srpearce: This is the first parse error reported as:
```
lms/templates/registration/activate_account_notice.html: 12:64: python-parse-error:                         "There's just one more step: Before you
```
You can see that the original code has a double-quote at the beginning and end of each line, but the new code is wrapped much earlier and is missing beginning and ending double-quotes on alternate lines.